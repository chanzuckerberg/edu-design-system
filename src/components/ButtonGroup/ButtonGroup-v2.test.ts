import { generateSnapshots } from '@chanzuckerberg/story-utils';
import type { StoryFile } from '@storybook/testing-react';
import * as stories from './ButtonGroup-v2.stories';

describe('<ButtonGroup /> (v2)', () => {
  generateSnapshots(stories as StoryFile);
});
