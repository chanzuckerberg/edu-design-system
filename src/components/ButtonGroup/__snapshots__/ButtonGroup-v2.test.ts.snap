// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ButtonGroup /> Default story renders snapshot 1`] = `
<div
  class="p-8"
>
  <div
    class="button-group button-group--horizontal"
  >
    <button
      class="button button--layout-none button--primary button--lg button--variant-default"
      type="button"
    >
      <span
        class="button__text"
      >
        Button 1
      </span>
    </button>
    <button
      class="button button--layout-none button--secondary button--lg button--variant-default"
      type="button"
    >
      <span
        class="button__text"
      >
        Button 2
      </span>
    </button>
  </div>
</div>
`;

exports[`<ButtonGroup /> HorizontalProgressive story renders snapshot 1`] = `
<div
  class="p-8"
>
  <div
    class="button-group button-group--horizontal-progressive"
  >
    <button
      class="button button--layout-none button--primary button--lg button--variant-default"
      type="button"
    >
      <span
        class="button__text"
      >
        Button 1
      </span>
    </button>
    <button
      class="button button--layout-none button--secondary button--lg button--variant-default"
      type="button"
    >
      <span
        class="button__text"
      >
        Button 2
      </span>
    </button>
  </div>
</div>
`;

exports[`<ButtonGroup /> Vertical story renders snapshot 1`] = `
<div
  class="p-8"
>
  <div
    class="button-group button-group--vertical"
  >
    <button
      class="button button--layout-none button--primary button--lg button--variant-default"
      type="button"
    >
      <span
        class="button__text"
      >
        Button 1
      </span>
    </button>
    <button
      class="button button--layout-none button--secondary button--lg button--variant-default"
      type="button"
    >
      <span
        class="button__text"
      >
        Button 2
      </span>
    </button>
  </div>
</div>
`;

exports[`<ButtonGroup /> WithFiveButtons story renders snapshot 1`] = `
<div
  class="p-8"
>
  <div
    class="button-group button-group--horizontal"
  >
    <button
      class="button button--layout-none button--primary button--lg button--variant-default"
      type="button"
    >
      <span
        class="button__text"
      >
        Button 1
      </span>
    </button>
    <button
      class="button button--layout-none button--primary button--lg button--variant-default"
      type="button"
    >
      <span
        class="button__text"
      >
        Button 2
      </span>
    </button>
    <button
      class="button button--layout-none button--primary button--lg button--variant-default"
      type="button"
    >
      <span
        class="button__text"
      >
        Button 3
      </span>
    </button>
    <button
      class="button button--layout-none button--primary button--lg button--variant-default"
      type="button"
    >
      <span
        class="button__text"
      >
        Button 4
      </span>
    </button>
    <button
      class="button button--layout-none button--primary button--lg button--variant-default"
      type="button"
    >
      <span
        class="button__text"
      >
        Button 5
      </span>
    </button>
  </div>
</div>
`;
